pinecone/core/grpc/index_grpc.py

```diff
@@ -207,6 +207,32 @@ def parse_stats_response(response: dict):
     return DescribeIndexStatsResponse(namespaces=namespace_summaries, dimension=dimension, _check_type=False)
 
 
+class PineconeGrpcFuture:
```

Comment:
Looks like you're missing the exception() and traceback() methods from the interface - https://github.com/grpc/grpc/blob/master/src/python/grpcio/grpc/framework/foundation/future.py#L163